pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -481,20 +461,6 @@ public long getValueOfTableGauge(final String tableName, final G gauge) {
     return gaugeValue == null ? 0 : gaugeValue.get();
   }
 
-  /**
-   * Gets the value of a table partition gauge.
-   *
-   * @param tableName The table name
-   * @param partitionId The partition name
-   * @param gauge The gauge to use
-   */
-  public long getValueOfPartitionGauge(final String tableName, final int partitionId, final G gauge) {
-    final String fullGaugeName = composeTableGaugeName(tableName, String.valueOf(partitionId), gauge);
-
-    AtomicLong gaugeValue = _gaugeValues.get(fullGaugeName);
-    return gaugeValue == null ? -1 : gaugeValue.get();
```

Comment:
@jugomezv Is there a particular reason why we put the default value as `-1`, not `0`?

Reply:
This getValueOfPartitionGauge is only used in unit tests and integration tests, as long as we don't break those tests, it should be fine even if we don't return -1 in the new method

Reply:
So should this be Visible for testing only?

Reply:
This code is removed. So no way to add `@VisibleForTesting` 😄